server/remark-includes.ts

```diff
@@ -197,8 +197,11 @@ export function resolveParamValue(val: string): string {
     );
   }
   let newVal = val.slice(1, val.length - 1);
+  const escapedQuoteRegexp = new RegExp('\\\\"', "g");
   if (val[0] == '"') {
-    newVal = newVal.replaceAll('\\"', '"');
+    newVal = newVal.replace(escapedQuoteRegexp, (escapedQuote) => {
```

Comment:
you shouldn't need to pass a function here if we are always returning the same value of `'"'`. Because your regex includes `g` in it, `replace` will work for all instances. I believe `newVal.replace(escapedQuoteRegexp, '"')` would work just fine. Same with the other. Is that not the case?

Reply:
I misremembered the types of `replace` args that triggered a replacement of all instances and somehow thought you needed both a regexp and a function. I've changed this in 1cdd986569a44c52170be2950a9b4b1debb128ed.

Reply:
Although, to your original point: I did notice that vercel has our Docs repo at node 14. I wanted to switch to 16 but have been a bit scared to push the button. Let's talk about doing this sooner than later. FWIW, I've been running docs locally on 16 and no hiccups but you never know.

Reply:
That sounds good to me! I've opened an issue here: https://github.com/gravitational/docs/issues/181